bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageHandler.java

```diff
@@ -133,20 +143,26 @@ private void pollStatus() throws IOException {
                     new QuantityType<>(controllerVariables.dist, MetricPrefix.CENTI(SIUnits.METRE)));
             if (controllerVariables.door == 0) {
                 updateState(OpenGarageBindingConstants.CHANNEL_OG_STATUS, OnOffType.OFF);
```

Comment:
small nit pick, would maybe  a switch statement be cleaner here?

```java
switch(controllerVariables.door){
    case 0:
        updateState(OpenGarageBindingConstants.CHANNEL_OG_STATUS, OnOffType.OFF);
        updateState(OpenGarageBindingConstants.CHANNEL_OG_STATUS_SWITCH, OnOffType.OFF);
        updateState(OpenGarageBindingConstants.CHANNEL_OG_STATUS_ROLLERSHUTTER, UpDownType.DOWN);
        updateState(OpenGarageBindingConstants.CHANNEL_OG_STATUS_CONTACT, OpenClosedType.CLOSED);
        break;
    case 1:
        updateState(OpenGarageBindingConstants.CHANNEL_OG_STATUS, OnOffType.ON);
        updateState(OpenGarageBindingConstants.CHANNEL_OG_STATUS_SWITCH, OnOffType.ON);
        updateState(OpenGarageBindingConstants.CHANNEL_OG_STATUS_ROLLERSHUTTER, UpDownType.UP);
        updateState(OpenGarageBindingConstants.CHANNEL_OG_STATUS_CONTACT, OpenClosedType.OPEN);
        break;
    default:
        logger.warn("Received unknown door value: {}", controllerVariables.door);
}
```

And of course the same for the vehicle state below.

Reply:
I was just looking at this and thinking the same thing :)

Reply:
Fixed in latest commit